src/08-time-tracking/time-tracking.apib

```diff
@@ -104,6 +104,9 @@ Get information about tracked time.
                         + ticket
                 + id: `5883723a-b5aa-4c9d-a02b-ec0dd25f6ca2` (string)
             + invoiceable: true (boolean)
+            + locked: true (boolean) - If true, the time tracking entry's owner cannot update it anymore
```

Comment:
Q: not sure if this comment fully reflects how the time tracking can be treated. If the owner of a locked TT is an admin, he/she can still update it, right?

PP: limit the comment to explaining what a locked time tracking is. I.e. the freeze time window has passed since creating it.